frontend/server/src/Cache.php

```diff
@@ -204,6 +204,7 @@ class Cache {
     const ADMIN_SCOREBOARD_PREFIX = 'scoreboard-admin-';
     const CONTESTANT_SCOREBOARD_EVENTS_PREFIX = 'scoreboard-events-';
     const ADMIN_SCOREBOARD_EVENTS_PREFIX = 'scoreboard-events-admin-';
+    const CODERS_OF_THE_MONTH = 'schools-of-the-month';
```

Comment:
```suggestion
    const CODERS_OF_THE_MONTH = 'coders-of-the-month';
```

si no, va a hacer conflicto con escuelas :S